layers/meta-opentrons/recipes-robot/opentrons-jupyter-notebook/files/jupyter-notebook.service

```diff
@@ -11,6 +11,8 @@ SupplementaryGroups=dialout
 StateDirectory=jupyter jupyter/notebooks jupyter/data jupyter/config ipython
 RuntimeDirectory=jupyter
 Environment=RUNNING_ON_VERDIN=true
+Environment=OT_SYSTEM_VERSION=
```

Comment:
nit: when doing this sed replacement style i kind of like having a placeholder value to replace, since this looks like it's trying to explicitly clear the system version. Can we make it like `OT_SYSTEM_VERSION=@@OT_SYSTEM_VERSION@@` or something similar and replace that?